cmake/GzConfigureProject.cmake

```diff
@@ -1,16 +1,16 @@
 #.rst
-# IgnConfigureProject
+# GzConfigureProject
 # -------------------
 #
 # gz_configure_project([VERSION_SUFFIX <pre|alpha|beta|etc>])
 #
 # Sets up a Gazebo library project.
 #
-# NO_IGNITION_PREFIX: Optional. Don't use ignition as prefix in
+# NO_GZ_PREFIX: Optional. Don't use Gazebo as prefix in
```

Comment:
We could go more generic with `NO_PROJECT_PREFIX`

Also, say "Don't use *gz-* as prefix in", since the prefix is `gz` not `Gazebo`

Reply:
https://github.com/gazebosim/gz-cmake/pull/275/commits/141d2aeb30d48f06b79f23f7cd9bef56d76c6e07